test/e2e/fixture/http.go

```diff
@@ -27,7 +28,14 @@
 		return nil, err
 	}
 	req.AddCookie(&http.Cookie{Name: common.AuthCookieName, Value: token})
-	return http.DefaultClient.Do(req)
+
+	httpClient := &http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
This code is part of the HTTP test fixture and is only being executed in test code so this warning should be safe to ignore. Additionally, this code is consistent with how the Argo CLI executes HTTP requests in test code as well (since the test fixture adds the insecure flag to every execution of the CLI).

Reply:
Just so the change has the smallest possible footprint, would it be advisable to toggle InsecureSkipVerify on `IsRemote()`?

Reply:
Good point. Updated logic to toggle InsecureSkipVerify on `IsRemote()`.